test-resources/rewriter-packs/custom/with-different-metas.yaml

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Objectionary.com
+# SPDX-License-Identifier: MIT
+---
+input: Q -> [[ x -> "hello", y -> "world" ]]
+output: |-
+  Φ ↦ ⟦
+    z ↦ "hello world"
+  ⟧
+rules:
+  custom:
+    - name: concat with metas
+      pattern: '[[ x -> !e_hello, y -> !e_world ]]'
+      result: '[[ z -> !e_hello_world ]]'
+      where:
+        - meta: '!d-space'
+          function: dataize
+          args: ['" "']
+        - meta: '!d_HELLO'
+          function: dataize
+          args: ['!e_hello']
+        - meta: '!d-WORLD1'
+          function: dataize
+          args: ['!e_world']
+        - meta: '!e_hello_world'
```

Comment:
@maxonfjvipon maybe you can also try `𝑒-hello` syntax

Reply:
@yegor256 it's also supported, check `!d-space` before